packages/engine/src/actions/link.test.js

```diff
@@ -69,13 +81,7 @@ test('Link with string pageId params', async () => {
   });
   const button = context._internal.RootBlocks.map['block:root:button:0'];
   const res = await button.triggerEvent({ name: 'onClick' });
-  expect(lowdefy._internal.link.mock.calls).toEqual([
```

Comment:
This should not change. We need to be using the real implementation of link here, that changes `'pageId'` to `{ pageId: 'pageId' }`